src/main/resources/templates/layout.html

```diff
@@ -2,10 +2,10 @@
 <html lang="en" xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout">
 <head>
     <meta charset="UTF-8">
-    <title>Title</title>
+    <title>Labor Helper</title>
+    <!Adding css to other pages>
```

Comment:
This comment isn't needed, as the purpose is obvious from context.  We don't put many comments in HTML.